src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesAsrAzureToAzureDiskReplicationConfig.md

```diff
@@ -64,6 +65,36 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DiskEncryptionSecretUrl
```

Comment:
Could you write an example in this file to showcase the new parameters?

Reply:
Added Example